apps/web/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "noEmit": true,
     "outDir": "../../out/apps/web",
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
reading the docs it looks like `node` is probably not what we want here, but I would have expected a new value of `nodenext` or `node16` instead. Does something in our codebase or this PR require `bundler`?